modules/API/pages/gsql-endpoints.adoc

```diff
@@ -0,0 +1,4415 @@
+----
+--
+====
+
+==== Parameters:
+
+None
+
+'''
+=== show a local vertex
+
+`GET /gsql/v1/schema/vertices/{vertexName}`
+
+This endpoint is used to show a local vertex.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X GET "http://localhost:14240/gsql/v1/schema/vertices/house?graph=g"
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"","results":{"Config":{"STATS":"OUTDEGREE_BY_EDGETYPE"},"IsLocal":true,"Attributes":[{"AttributeType":{"Name":"STRING"},"AttributeName":"name"}],"PrimaryId":{"AttributeType":{"Name":"UINT"},"AttributeName":"id"},"Name":"house"}}
+----
+--
+====
+
+==== Parameters:
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `graph`
+| yes
+| the vertices on which graph.
+|===
+
+'''        
+=== show a global vertex
+
+`GET /gsql/v1/schema/vertices/{vertexName}`
+
+This endpoint is used to show a global vertex
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X GET "http://localhost:14240/gsql/v1/schema/vertices/person"
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"","results":{"Config":{"STATS":"OUTDEGREE_BY_EDGETYPE"},"Attributes":[{"AttributeType":{"Name":"UINT"},"AttributeName":"sid"},{"AttributeType":{"Name":"STRING"},"AttributeName":"name"}],"PrimaryId":{"AttributeType":{"Name":"UINT"},"AttributeName":"id"},"Name":"person"}}
+----
+--
+====
+
+==== Parameters:
+
+NONE
+
+'''     
+
+=== create vertex usgin gsql command 
+
+`POST /gsql/v1/schema/vertices`
+
+This endpoint is used to create global vertices using gsql command

```

Comment:
Need to specify we accept query string as the payload, and the payload is the gsql command